### ISSUE TITLE
Lerna.json options not being applied

### ISSUE THREAD

**Issue by hainessss** [intuit/auto#1507]
The `auto shipit` command is logging this in my CI:

```
✔  success   Wrote authentication token string to /root/.npmrc
Error: Running command 'npx' with args [lerna, publish, --yes, from-package, --registry, https://registry.yarnpkg.com/] failed
```

despite having this configs:

.autorc
```
{
  "plugins": [
    ["npm", { "setRcToken": false }]
  ]
}
```
lerna.json
```
{
  "version": "independent",
  "npmClient": "yarn",
  "useWorkspaces": true,
  "command": {
    "publish": {
      "registry": "https://privateregistry.jfrog.io/artifactory/api/npm",
      "ignoreChanges": [
        "dist",
        "*.md",
        "*.proof.ts",
        "*.test.tsx",
        "*.stories.tsx",
        "*.tsbuildinfo"
      ]
    }
  }
}
```

Did something change in a recent version where I need to define these options somewhere else?

**Comment by hipstersmoothie** [comment]
https://github.com/intuit/auto/pull/1412

I didn't account for setting the registry in that place. can you move it to the root package.json instead? I would also take a PR to add finding the registry here too. Won't have time to do this for a few days

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.53.0`](https://github.com/intuit/auto/releases/tag/v9.53.0) :rocket:
